BuildcsprojtoMem/Program.cs

```diff
@@ -78,13 +78,13 @@ public static void MyMethod(string CsProjPath)
             Console.Write("Assembly:");
             if (project.Properties.Any(n => n.Name == "TargetPath"))
             {
-                var mypath = System.Reflection.Assembly.GetEntryAssembly().Location;
-                var targetPath = project.GetProperty("TargetPath");
-                var targetPathString = targetPath.EvaluatedValue.ToString();
-                var assembly = Assembly.LoadFrom(targetPathString);
-                foreach (AssemblyName assemblyName in assembly.GetReferencedAssemblies())
+                var MyPath = System.Reflection.Assembly.GetEntryAssembly().Location;
```

Comment:
variable names start with lowercase, 'var myPath'

Reply:
[This ](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md)looks like a pretty good primer